projects/Mallard/src/helpers/settings/defaults.ts

```diff
@@ -52,8 +52,9 @@ export const defaultSettings: Settings = {
     notificationServiceRegister: __DEV__
         ? notificationServiceRegister.code
         : notificationServiceRegister.prod,
-    zipUrl: apiUrl + 'zips/',
+    zipUrl: apiUrl + 'zips',
     cacheClearUrl: apiUrl + 'cache-clear',
+    appPrefix: 'daily-edition',
```

Comment:
If this app was to display a daily edition and a weekly edition would this prevent that?

Reply:
I think our whole data model in our app would prevent it.